src/themes/greyVest/index.js

```diff
@@ -350,7 +350,11 @@ export let GVStyle = () => (
         /*background: #f6f6f6;*/
       }
       .gv-search-bar .gv-box {
-        padding: 0;
+        padding: 8px 10px;
```

Comment:
Hope this is alright -- it was bothering me to see the search input so close to the left with all that vertical space, so I snuck in a bit of padding. It still looks nice with a few rows of entered tags (I checked) 🙂

Reply:
If it looks nicer then better :)